great-idea/css/index.css

```diff
@@ -57,10 +57,249 @@ html, body {
     font-family: 'Titillium Web', sans-serif;
 }
 
+/*
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 15px;
+	margin-bottom: 15px;
+	
+}
+*/
+
+/* Your code starts here! */
+.container {
+	width: 900px; 
+	margin: auto;
+	
```

Comment:
So, something to note here...

![image](https://user-images.githubusercontent.com/8473052/57054935-be32ae80-6c65-11e9-8882-12111d5af5a6.png)

When I reduce the screen width of my browser, it ends up cutting off the content and creating a horizontal scrollbar due to the explicit width being defined here. One way to make this more responsive is by using a `max-width` instead.

For instance:
```css
.container {
  width: 100%;
  max-width: 900px;
  margin: 0 auto;
}
```